lib/strings.md

```diff
@@ -5,20 +5,20 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A String is a variable that stores a sequences of characters.
```

Comment:
Instead of "variable", I'd encourage you to think of Strings as Ruby "objects".

You can always store a String in a variable, eg: `name = "Colin"`